Ginger/GingerCore/ALM/QtestCore.cs

```diff
@@ -0,0 +1,432 @@
+                    {
+                        QTestApiModel.TestStepResource stepResource = new QTestApiModel.TestStepResource(   null, null, 
+                                                                                                            ((Activity)actIdent.IdentifiedActivity).Description,
+                                                                                                            ((Activity)actIdent.IdentifiedActivity).Expected);                 
+                        stepResource.PlainValueText = ((Activity)actIdent.IdentifiedActivity).ActivityName;                                                                   
+                        testcaseApi.AddTestStep((long)Convert.ToInt32(ALMCore.AlmConfig.ALMProjectKey), testCase.Id, stepResource);
+                    }
+                    activitiesGroup.ExternalID = testCase.Id.ToString();
+                    activitiesGroup.ExternalID2 = testCase.Id.ToString();
+                }
+                else //##update existing test case
+                {
+                    ////##update existing test case
+                    //test = ImportFromQC.GetQCTest(activitiesGroup.ExternalID);
+
+                    ////delete the un-needed steps
+                    //DesignStepFactory stepF = test.DesignStepFactory;
+                    //List stepsList = stepF.NewList("");
+                    //foreach (DesignStep step in stepsList)
+                    //{
+                    //    if (activitiesGroup.ActivitiesIdentifiers.Where(x => x.IdentifiedActivity.ExternalID == step.ID.ToString()).FirstOrDefault() == null)
+                    //        stepF.RemoveItem(step.ID);
+                    //}
+
+                    ////delete the existing parameters
+                    //StepParams testParams = test.Params;
+                    //if (testParams.Count > 0)
+                    //{
+                    //    for (int indx = 0; indx < testParams.Count; indx++)
+                    //    {
+                    //        testParams.DeleteParam(testParams.ParamName[indx]);
+                    //        testParams.Save();
+                    //    }
+                    //}
+                }
+
+                return true;
+            }
+            catch (Exception ex)
+            {
+                result = "Unexpected error occurred- " + ex.Message;
+                Reporter.ToLog(eLogLevel.ERROR, "Failed to export the " + GingerDicser.GetTermResValue(eTermResKey.ActivitiesGroup) + " to QC/ALM", ex);
+                return false;
+            }
+
+
+            return true;
+        }
+
+        public bool ExportBusinessFlowToALM(BusinessFlow businessFlow, QtestTestSuite mappedTestSuite, string parentObjectId, ObservableList<ExternalItemFieldBase> testSetFields, ref string result)
+        {
+            // return ExportToQC.ExportBusinessFlowToQC(businessFlow, mappedTestSuite, uploadPath, testSetFields, ref result);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)